lib/avatar/data/image_merger.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/painting.dart';
+import 'package:flutter/material.dart';
+import 'dart:ui' as ui;
+import 'dart:async';
+
+class ImageMerger extends StatelessWidget {
```

Comment:
A class comment would be helpful. You should also rename this class to make it more obvious that it is a widget.